api/src/main/java/datawave/webservice/response/objects/DefaultKey.java

```diff
@@ -43,34 +43,34 @@ public DefaultKey() {
     
     @Override
     public String getRow() {
-        if (this.row.getType().equals(TypedValue.XSD_STRING) && this.row.isBase64Encoded() == true) {
-            return new String(Base64.decodeBase64(this.row.getValue().toString().getBytes(Charset.forName("UTF-8"))));
+        if (this.row.getType().equals(TypedValue.XSD_STRING) && this.row.isBase64Encoded()) {
+            return new String(Base64.decodeBase64(this.row.getValue().toString().getBytes(StandardCharsets.UTF_8)), StandardCharsets.UTF_8);
```

Comment:
The modernizer-maven-plugin is also good at catching some of these string-based charsets, and suggesting the concrete type/instance, as well as identifying lots of other code-modernizations that can be done with newer versions of Java. It's definitely worth checking out, if you want another tool beyond spotbugs to help clean up internal tech-debt.

Reply:
Will take a look. Thanks!